src/components/NavBar/style.ts

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { sans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 78px;
+  background: ${COLORS.blueMid};
+  z-index: 1000;
+  box-shadow: 0px 4px 7px 0px rgba(0, 0, 0, 0.1);
+  justify-content: space-between;
+`;
+
+export const NavBarSectionDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 18px;
+  margin-left: 34px;
+  margin-right: 34px;
+`;
+
+export const AuthButtons = styled.div`
+  display: flex;
+  gap: 1rem;
+  align-self: center;
+  top: 0;
+  right: 0;
+`;
+export const NoUnderlineLink = styled(LinkColored)<{ $isActive: boolean }>`
```

Comment:
we can remove the `$isActive` prop since it's no longer used

Reply:
also nit: can we add a newline before this so we can separate the definition of `AuthButtons` from that of `NoUnderlineLink`?